Core/DisconnectMeContentBlockerParser.swift

```diff
@@ -66,10 +68,9 @@ public struct DisconnectMeTrackersParser {
         return host.replacingOccurrences(of: "www.", with: "")
     }
     
-    func isSupported(categoryName: String) -> Bool {
-        guard let category = Category.init(rawValue: categoryName) else { return false }
-        return category == .advertising || category == .analytics || category == .social
+    func category(name: String, isIn categories: [Category]) -> Bool {
+        guard let category = Category.init(rawValue: name) else { return false }
```

Comment:
This is a valid, but slightly weird way of instantiating something.  You can use:

`Category(rawValue: name)`

Reply:
Eeek, that's a bad autocomplete habit. I cleared all of them, thanks!